src/features/landing/components/ForWhoSection/ForWhoSection.tsx

```
@@ -1,26 +1,21 @@
 import { Box, Container } from '@mui/material';
-import { ImageProps } from 'next/image';
-import { getOptimizedImageProps } from 'next-export-optimize-images/image';
 import React from 'react';
 
-import DesktopVectorIcon from '../../assets/img/about-vilna/FrameDesktop.png';
-import TabletVectorIcon from '../../assets/img/about-vilna/FrameTablet.png';
-
 import { Cards } from './Cards';
 import MainTitle from './MainTitle/MainTitle';
 import styles from './styles';
+import bigScreen from '../../assets/img/about-vilna/mobileScreen.png';
```

ESLint

[error] `../../assets/img/about-vilna/mobileScreen.png` import should occur after import of `../../assets/img/about-vilna/mobile2Screen.png`

(import/order)